verification/verify_test.go

```diff
@@ -300,9 +300,9 @@ func TestVerifyPixels(t *testing.T) {
 
 	// Test no writing temp file with incorrect pixels
 	err = verifyPixels("../server/test.flv", nil, 50)
-	assert.Equal(ErrPixelMismatch, err)
+	assert.EqualError(err, "Invalid data found when processing input")
```

Comment:
We can update the comment for this test since `countPixels()` always writes a temp file. Maybe something like "Test nil data"

Reply:
done in e2942c57d4fba7fe78fde65be183a53668c38da8